pkg/core/database/heavy/transactions.go

```diff
@@ -194,7 +194,18 @@ func (t transaction) StoreBlock(b *block.Block) error {
 	value = b.Header.Hash
 	t.put(key, value)
 
-	return nil
+	// Delete expired bid values
+	key = BidValuesPrefix
+	iterator := t.snapshot.NewIterator(util.BytesPrefix(key), nil)
+
+	for iterator.Next() {
+		height := binary.LittleEndian.Uint64(iterator.Key()[1:])
+		if height < b.Header.Height {
+			t.batch.Delete(iterator.Key())
+		}
+	}
+
+	return iterator.Error()
```

Comment:
What kind of errors could be expected at that point?

Reply:
No idea, but I figured it would be good practice to check just in case.